exercise/static/exercise/chapter.js

```diff
@@ -859,10 +856,7 @@
     },
 
     renderMath: function() {
-      if (typeof window.MathJax === "undefined") {
-        return;
-      }
-      window.MathJax.Hub.Queue(["Typeset", window.MathJax.Hub, this.element.get(0)]);
+      $.fn.aplusTypesetMath(this.element.get(0))
```

Comment:
```suggestion
      this.element.aplusTypesetMath();
```
Something like that.